servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpRequestMetaData.java

```diff
@@ -406,6 +406,27 @@ default boolean hasQueryParameter(final String key) {
      */
     boolean removeQueryParameters(String key, String value);
 
+    /**
+     * Returns the <a href="https://datatracker.ietf.org/doc/html/rfc3986#section-3.5">fragment</a> part of the request target.
+     *
+     * @return the fragment part of the request target.
+     */
+    @Nullable
+    default String fragment() {
```

Comment:
Forgot to ask, please add the following comment here:
```
// FIXME: 0.43 - consider removing default implementations
```
It's a reminder for us when it's expected to have some API changes.